central/policycategory/search/searcher_impl.go

```diff
@@ -0,0 +1,107 @@
+package search
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/central/policycategory/index"
+	categoryMapping "github.com/stackrox/rox/central/policycategory/index/mappings"
+	"github.com/stackrox/rox/central/policycategory/store"
+	"github.com/stackrox/rox/central/role/resources"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/blevesearch"
+	"github.com/stackrox/rox/pkg/search/paginated"
+	"github.com/stackrox/rox/pkg/search/sortfields"
+)
+
+var (
+	defaultSortOption = &v1.QuerySortOption{
```

Comment:
Default sort options should be moved to the service layer and not in the searcher layer

Reply:
A quick search in the code base shows default options in searcher in almost all entities. Also makes sense since it is used in the searcher. Prefer leaving this as is, unless there is a strong reason to not do so.

Reply:
Correct, this is the beginning of a change that we should start making. Sorting unnecessarily makes the queries more expensive and many internal use cases don't care that the data is sorted. We should rely on the caller to request that the data be sorted and so the use case of a default sort should be applied in the service layer as it is handling the response to the client

Reply:
Ok then let's do a separate pass on this. In this particular case, I wouldn't worry since scale isn't an issue.

Reply:
fair enough